.circleci/PACKAGE_LOCK_CHECKSUMS

```diff
@@ -0,0 +1,27 @@
+4057861937e941f6ce5085526528c39c  ./package-lock.json
```

Comment:
Now you have two places to add changes in modules. I personally think this is unmaintainable.